webdriver/support/merge_dictionaries.py

```diff
@@ -0,0 +1,42 @@
+def merge_dictionaries(first, second):
```

Comment:
I’d expose this in this module’s `__init__.py` so that we don’t end up having `from support.merge_dictionaries import merge_dictionaries`, but rather `from support import merge_dictionaries`.